src/index.html

```diff
@@ -1,27 +1,35 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
-    <title>Wheel of Wizards</title>
+    <title> Wheel of Wizards </title>
   </head>
   <body>
-    <h2>Welcome to Wheel of Wizards</h2>
-    <form class="splash" action="">
-      <label for="player1">Player 1</label>
-      <input id="player1Input" class="nameInputs" name="player1" type="text" placeholder="Enter Player One Name">
-      <label for="player2">Player 2</label>
-      <input id="player2Input" class="nameInputs" name="player2" type="text" placeholder="Enter Player Two Name">
-      <label for="player3">Player 3</label>
-      <input id="player3Input" class="nameInputs" name="player3" type="text" placeholder="Enter Player Three Name">
-      <button class="startGame">Start Game</button>
-    </form>
+    <section class='welcome-section'>
```

Comment:
Lets chat and maybe even ask about using kabob vs camel case. I think it is best if we are consistent with this.